content/02.introduction.md

```diff
@@ -56,6 +54,8 @@ In the past two decades, however, highly pathogenic human coronaviruses have bee
 The current pandemic of COVID-19, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), represents an acute and rapidly spreading global health crisis with symptoms for reported cases ranging from mild to severe or fatal [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html] and including outcomes such as acute respiratory distress, acute lung injury and other pulmonary complications.
 The possibility of asymptomatic cases is also being investigated.
```

Comment:
I think it is clearer now that asymptomatic spread does occur, estimated in the range of 10-30% of cases. Here are some relevant references: 
10.1056/NEJMc2001468
10.1101/2020.05.10.20097543
10.1016/S0140-6736(20)31483-5

There is also what is being termed presymptomatic spread in 30-50% of cases:
10.1073/pnas.2008373117 
10.1101/2020.05.10.20097543

Possibly we could be more forward with that :)

Reply:
Shall we open a new pull request for those changes after closing this one? What does everyone think?

Reply:
@SiminaB sounds good to me!  #531